index.html

```diff
@@ -296,7 +296,7 @@ <h2>StackStorm connects all your apps, services, and workflows.</h2>
 <!------------------- LOGOS ------------------->
 <div class="full-width border-bottom shadow-bottom">
 <div class="content-container">
-<h3>Open source and trusted by the enterprise</h3>
+<h3>Free & open source and trusted by the enterprise</h3>
```

Comment:
Alternatives to reinforce the statement even more could be also: 
```suggestion
<h3>Always free & open source and trusted by the enterprise</h3>
```